frontend/src/validation-schemas/episode/create-episode/create-podcast.validation-schema.ts

```diff
@@ -0,0 +1,9 @@
+import { Joi } from 'helpers/helpers';
```

Comment:
frontend/src/validation-schemas/episode/create-episode/create-podcast.validation-schema.ts
This is probably a mistake in the schema file name. I think there must be:
frontend/src/validation-schemas/episode/create-episode/create-episode.validation-schema.ts